include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -31,7 +31,7 @@ def LowerFIRRTLAnnotations : Pass<"firrtl-lower-annotations", "firrtl::CircuitOp
     Option<"noRefTypePorts", "no-ref-type-ports", "bool", "false",
       "Create normal ports, not ref type ports.">,
   ];
-  let dependentDialects = ["hw::HWDialect"];
+  let dependentDialects = ["circt::sv::SVDialect", "circt::hw::HWDialect"];
```

Comment:
Yeah, don't we already generate SV operations sometimes? Once this is settled re:where things are going / output / functionality, consider breaking off this sort of thing separately.